package.json

```diff
@@ -20,23 +20,24 @@
   },
   "homepage": "https://github.com/FormidableLabs/radium-docs#readme",
   "devDependencies": {
-    "builder-docs-archetype-dev": "^2.0.1"
+    "builder-docs-archetype-dev": "^3.0.0"
   },
   "dependencies": {
     "babel-preset-stage-1": "^6.5.0",
     "builder": "^2.9.1",
-    "builder-docs-archetype": "^2.0.1",
+    "builder-docs-archetype": "^3.0.2",
     "chai": "^3.2.0",
     "color": "^0.11.3",
-    "ecology": "^1.5.2",
-    "formidable-landers": "^3.1.0",
+    "ecology": "^1.6.1",
+    "formidable-landers": "^4.0.4",
     "history": "~1.17.0",
     "markdown-it": "^7.0.0",
     "markdown-it-named-headers": "0.0.4",
     "prismjs": "^1.5.1",
     "radium": "*",
-    "react": "15.2.1",
-    "react-dom": "15.2.1",
+    "react": "^15.0.1",
+    "react-addons-test-utils": "^15.0.1",
+    "react-dom": "^15.0.1",
```

Comment:
😬 mine too. I matched it to the react dependency in `radium`, so they’re both matchy-matchy.